src/oxli/hashgraph.cc

```diff
@@ -828,14 +828,16 @@ const
     while(!kmers.done()) {
         n++;
         if (n % 10000 == 0) {
-            std::cout << "... find_high_degree_nodes: " << n << "\n";
-            std::cout << std::flush;
+            std::cout << "\r... find_high_degree_nodes: " << n;
         }
         Kmer kmer = kmers.next();
         if ((traverser.degree(kmer)) > 2) {
```

Comment:
So, depending on how we want to define a cDBG, this could introduce a bug. Consider:
```
(HDN)--->(neighbor_A)---...
     \-->(neighbor_C)---...
```
This is a degree 2 node, but it should probably be counted as a HDN in the context of a cDBG (in the context of my streaming compactor, I'm counting it as one).

Maybe this deserves its own issue/thread. Thoughts?

Reply:
Yep! Punting to issue #1809.